src/web_app/js/appcontroller_test.js

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
```

Comment:
Right, that's generally the practice: https://chromium.googlesource.com/chromium/src/+/master/styleguide/c++/c++.md#File-headers
"$YEAR should be set to the current year at the time a file is created, and not changed thereafter."

It makes life easier for everyone, especially after new year's eve.